AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginAuthCognitoTests/AWSDataStoreAuthBaseTest.swift

```diff
@@ -203,41 +196,43 @@ extension AWSDataStoreAuthBaseTest {
     /// - Parameter user
     func signIn(user: TestUser?,
                 file: StaticString = #file,
-                line: UInt = #line) {
+                line: UInt = #line) async throws {
         guard let user = user else {
             XCTFail("Invalid user", file: file, line: line)
             return
         }
-        let signInInvoked = expectation(description: "sign in completed")
-        _ = Amplify.Auth.signIn(username: user.username,
-                                password: user.password, options: nil) { result in
-            switch result {
-            case .failure(let error):
-                XCTFail("Signin failure \(error)", file: file, line: line)
-                signInInvoked.fulfill() // won't count as pass
-            case .success:
-                signInInvoked.fulfill()
-            }
-        }
-        wait(for: [signInInvoked], timeout: TestCommonConstants.networkTimeout)
+//        let signInInvoked = expectation(description: "sign in completed")
+        _ = try await Amplify.Auth.signIn(username: user.username,
+                                password: user.password, options: nil)
```

Comment:
if you are having troubles confirming if the user is signed in or not, you can construct a dummy test like
```
func testSignIn() async throws {
   try await signIn(user: testUser1)
}
```

and within the `signIn` method, handle the return value by printing it out, and intercept the errors coming back instead of throwing it back up the chain
```
do {
  let response = try await Amplify.Auth.signIn(....)
print("Got response for signIn: \(response)")
} catch {
  print("Got error back on signIn \(error)")
}
```

Reply:
sure thing that was my next plan! but wanted to first make the code build successfully, so I made changes.